_posts/2020-02-25-password-encoding-spring-security.md

```diff
@@ -0,0 +1,535 @@
+---
+title: Password Encoding with Spring Security
+categories: [spring-boot]
+date: 2020-02-25 06:00:00 +1100
+modified: 2020-02-25 06:00:00 +1100
+author: artur
+excerpt: "When you have to store user passwords, you have to do securely. Let's have a look how to handle passwords with Spring Security in a Spring Boot application "
+image:
+  auto: 0062-password
```

Comment:
I changed this to `0063-password`. Can you add the URL to unsplash to `credits.yml`?

Reply:
done